PowerEditor/src/NppDarkMode.cpp

```diff
@@ -361,10 +362,26 @@ namespace NppDarkMode
 		_options = configuredOptions();
 
 		initExperimentalDarkMode();
+		initAdvancedOptions();
+
+		if (NppDarkMode::isWindowsModeEnabled())
+		{
+			NppParameters& nppParam = NppParameters::getInstance();
+			NppGUI& nppGUI = nppParam.getNppGUI();
+			nppGUI._darkmode._isEnabled = NppDarkMode::isDarkModeReg();
```

Comment:
change to `nppGUI._darkmode._isEnabled = NppDarkMode::isDarkModeReg() && !IsHighContrast();` to be sure